File: Money with Baba .c
#include<stdio.h>
#include <stdlib.h>

int main()
{
    int amountInHand, beggarCount;
    scanf("%d%d",&amountInHand,&beggarCount);
    int initialAmount=amountInHand;
    int counter=1;
    while(counter <= beggarCount){
        initialAmount = initialAmount*2;
        counter++;
    }
    
    printf("%d",initialAmount);
}

File: Money with Baba.java
import java.util.*;
public class Hello {

    public static void main(String[] args) {
	Scanner sc = new Scanner(System.in);
	 int amountInHand = sc.nextInt();
	 int beggarCount = sc.nextInt();
	 int initialAmount=amountInHand;
     int counter=1;
    
     while(counter <= beggarCount){
        initialAmount = initialAmount*2;
        counter++;
     }
     
     System.out.print(initialAmount);

	}
}
